crates/matrix-sdk/Cargo.toml

```diff
@@ -29,8 +29,8 @@ indexeddb_cryptostore = ["matrix-sdk-indexeddb/encryption", "encryption"]
 encryption = ["matrix-sdk-base/encryption"]
 qrcode = ["encryption", "matrix-sdk-base/qrcode"]
 # TODO merge those two sled features
-sled_state_store = ["matrix-sdk-sled"]
-sled_cryptostore = ["matrix-sdk-sled/encryption", "encryption"]
+sled_state_store = ["matrix-sdk-sled/state-store"]
+sled_cryptostore = ["matrix-sdk-sled/crypto-store", "encryption"]
```

Comment:
can we agree on one way to spell this (at least for features)? Either `-` or `_` and either one word or two? We now have `sled_cryptostore`, `sled_state_store`, `crypto-store` and `state-store`.

Reply:
Sure, I think I prefer two words with a `-` for the separator.